eclipse/plugin/src/nl/tudelft/watchdog/eclipse/logic/ui/listeners/staticanalysis/MarkerHolder.java

```diff
@@ -0,0 +1,51 @@
+package nl.tudelft.watchdog.eclipse.logic.ui.listeners.staticanalysis;
+
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.core.runtime.CoreException;
+import org.joda.time.DateTime;
+
+import nl.tudelft.watchdog.core.logic.ui.listeners.staticanalysis.StaticAnalysisMessageClassifier;
+
+/**
+ * Because an {@link IMarker} throws exceptions on {@link IMarker#getAttribute(String)} if it is removed,
```

Comment:
once it has been trashed by Eclipse ? (to pickup your wording from the other class)